### PR TITLE
[zh] Sync #13933 into Chinese

### PR DESCRIPTION
**Pull request by wilsonwu**
## Description

Sync #13933 into Chinese

## Reviewers

@windsonsea 

- [ ] Ambient
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure
- [x] Localization/Translation

### CODE REVIEW

File: content/zh/docs/ops/integrations/prometheus/index.md [hunk 1]
```diff
@@ -126,10 +126,24 @@ Chart 提供的配置。
 
 #### TLS 设置 {#TLS-settings}
 
-控制平面，网关和 Envoy Sidecar 指标将会作为明文收集。但是，应用程序指标将遵循
-Istio 为任何工作负载进行的配置。特别是如果启用了
-[Strict mTLS](/zh/docs/tasks/security/authentication/authn-policy/#globally-enabling-istio-mutual-tls-in-strict-mode)，
-则需要将 Prometheus 配置为使用 Istio 证书收集指标。
+控制平面，网关和 Envoy Sidecar 指标将会作为明文被收集。
+
```
Comment (windsonsea) [inline content/zh/docs/ops/integrations/prometheus/index.md:130]: The blank line 130 shall be removed

File: content/zh/docs/ops/integrations/prometheus/index.md [hunk 2]
```diff
@@ -126,10 +126,24 @@ Chart 提供的配置。
...
+但是，应用程序指标将遵循为工作负载配置的任何
+[Istio 身份验证策略](/zh/docs/tasks/security/authentication/authn-policy)。
+
+* 如果您使用 `STRICT` 模式，则需要将 Prometheus 配置为使用 Istio 证书进行抓取，如下所述。
+* 如果您使用 `PERMISSIVE` 模式，工作负载通常接受 TLS 和明文。
+  但是，Prometheus 无法发送 Istio 所需的 `PERMISSIVE` 模式所需的 TLS 特殊变体。
```
Comment (windsonsea) [inline content/zh/docs/ops/integrations/prometheus/index.md:136]:
```suggestion
  然而，Prometheus 无法发送 Istio 针对 `PERMISSIVE` 模式所需的 TLS 特殊变体。
```
`但是` 是不是语气太强烈了

File: content/zh/docs/ops/integrations/prometheus/index.md [hunk 3]
```diff
@@ -126,10 +126,24 @@ Chart 提供的配置。
...
+  因此，您不得在 Prometheus 中配置 TLS。
+* 如果使用 `DISABLE` 模式，则 Prometheus 不需要 TLS 配置。
```
Comment (windsonsea) [inline content/zh/docs/ops/integrations/prometheus/index.md:138]:
```suggestion
* 如果您使用 `DISABLE` 模式，则 Prometheus 不需要 TLS 配置。
```
keep the style consistent

File: content/zh/docs/ops/integrations/prometheus/index.md [hunk 4]
```diff
@@ -126,10 +126,24 @@ Chart 提供的配置。
...
+
+{{< tip >}}
+请注意，这仅适用于 Istio-terminated 的 TLS。如果您的应用程序直接处理 TLS：
```
Comment (windsonsea) [inline content/zh/docs/ops/integrations/prometheus/index.md:141]:
```suggestion
请注意，这仅适用于 Istio 终止的 TLS。如果您的应用程序直接处理 TLS：

```

File: content/zh/docs/ops/integrations/prometheus/index.md [hunk 5]
```diff
@@ -126,10 +126,24 @@ Chart 提供的配置。
...
+* 不支持 `STRICT` 模式，因为 Prometheus 需要发送两层 TLS，但它无法做到这一点。
```
Comment (windsonsea) [inline content/zh/docs/ops/integrations/prometheus/index.md:142]:
```suggestion
* 不支持 `STRICT` 模式，因为 Prometheus 将需要发送两层 TLS，但 Prometheus 无法做到这一点。
```
`它` 需要翻出来

File: content/zh/docs/ops/integrations/prometheus/index.md [hunk 6]
```diff
@@ -126,10 +126,24 @@ Chart 提供的配置。
...
+* `PERMISSIVE` 模式和 `DISABLE` 模式的配置应与 Istio 不存在时相同。
```
Comment (windsonsea) [inline content/zh/docs/ops/integrations/prometheus/index.md:143]:
```suggestion
* `PERMISSIVE` 模式和 `DISABLE` 模式应被配置为好似 Istio 不存在时的情形。
```

**Review by windsonsea — COMMENTED** [review]
Some text suggestions. Welcome to more discussions.

File: content/zh/docs/ops/integrations/prometheus/index.md (hunk 1 above)
Comment (wilsonwu) [reply content/zh/docs/ops/integrations/prometheus/index.md:130]: Done

**Review by windsonsea — APPROVED** [review]
Thanks.

**Pull request merged**